src/converter/internal/convertercontroller.cpp

```diff
@@ -181,7 +181,8 @@ mu::Ret ConverterController::convertPageByPage(INotationWriterPtr writer, INotat
     TRACEFUNC;
 
     for (size_t i = 0; i < notation->elements()->pages().size(); i++) {
-        const QString filePath = io::path_t(io::dirpath(out) + "/" + io::basename(out) + "-%1." + io::suffix(out)).toQString().arg(i + 1);
+       const QString filePath 
```

Comment:
Ah, I see why: you accidentally removed a space in front of this line

Reply:
By the way, if you click on the "Details" link for the code style check, it tells you exactly what's wrong.

Reply:
ok,I will pay attention to this

Reply:
At least I resolve the format problem😁, but the Generate and Compare part indicates that"This PR appears to change some of the visual tests".😐I think maybe some `basename` can't be changed to `completeBasename` due to the reason that it will cause a difference in UI appearance?

Reply:
I've looked at the differences and I think they are unrelated to the PR. Could you please try rebasing (`git pull --rebase upstream master`)? I expect that this will fix it. 

(Those visual tests only compare the engraving output, not screenshots of the UI, by the way.)